wl/server/HandleCommand.java

```diff
@@ -822,6 +817,26 @@ private void handleCmdSubmitScreenshot() throws Exception {
 					throw new ServerUtils.WLProtocolException("Checksum mismatch: expected " +
 					                                          cmd[3] + ", found " + checksum);
 				ServerUtils.checkEndOfStream(in);
+
+				String mimetype = Utils.bashOutput("mimetype", "--brief", file.getPath());
```

Comment:
Better use the additional parameter `-M`, or `--magic_only`, for mimetype:
~~~~
$:> mimetype --brief mini_theme_2.jpg
image/jpeg
$:> mv mini_theme_2.jpg mini_theme_2.png
$:> mimetype --brief mini_theme_2.png
image/png
$:> mimetype -M --brief mini_theme_2.png
image/jpeg
~~~~